custom/icds_reports/models/aggregate.py

```diff
@@ -1590,6 +1592,33 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateAvailingServiceForms(models.Model, AggregateMixin):
+    """ Aggregated data for the availing services
+
+    A availing services exists for each state_id
+
+    A row exists for every person case that has had a record of registration
+    submitted against it this month
```

Comment:
This is not true, right? There is one row for every person case that has EVER had an availing services form (not just cases registered this month).